pyuvsim/telescope.py

```diff
@@ -114,7 +110,6 @@ class BeamList:
     def __init__(
         self,
         beam_list=None,
-        uvb_params=None,
```

Comment:
I see that we're free ourselves from a workaround before but could this be being used by users too?
Should be be concerned with the API change and push for a new release?

Reply:
I think this was only used under the hood for tracking extra items on UVBeams that are no longer included now that we require the most recent pyuvdata version. We do have external testing against `hera_sim` which passed. @steven-murray do you have any concerns here?

We are planning a new release pretty soon, but if we're concerned about this as an API change we should have a period of deprecation before removing it.

Reply:
Currently all the code related to `uvb_params` is unreachable unless you add weird things to it by hand.

Reply:
okay that's fine I think :+1: